concept/type/impl/TypeImpl.java

```diff
@@ -114,7 +114,7 @@ public boolean isAbstract() {
     public abstract FunctionalIterator<? extends Type> getSubtypesExplicit();
 
     <THING> FunctionalIterator<THING> instances(Function<ThingVertex, THING> thingConstructor) {
-        return getSubtypes().flatMap(t -> graphMgr.data().get(t.vertex)).map(thingConstructor);
+        return getSubtypes().flatMap(t -> graphMgr.data().getAll(t.vertex )).map(thingConstructor);
```

Comment:
we had to rename this from `get` to `getAll` due to signatures clashing with the new optional booleans